app/js/tests/test_store_update_handling.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+var test = require( 'tape' ),
+	sinon = require( 'sinon' ),
+	storeUpdateHandling = require( '../lib/store_update_handling' )
+	;
+
+function createFakeStore( storeData ) {
+	return {
+		subscribe: sinon.spy(),
+		getState: sinon.stub().returns( storeData ),
+		// Call all callback functions that were registered through calls to `subscribe`
+		fakeUpdate: function () {
+			var subscribeCalls = this.subscribe.args;
+			subscribeCalls.forEach( function ( subscribeArguments ) {
+				// get the first argument to the `subscribe` call
+				var updateCallback = subscribeArguments[ 0 ];
+				// call the callback, using its own context
```

Comment:
Does this comment add any value? Assuming you know what `call` does in JS, this ought to be obvious no?